### ISSUE TITLE
Glue Generation fails, saying output directory does not exist when the directory does exist

### ISSUE THREAD

**Issue by BrainBlasted** [neikeq/GodotSharp#31]
```
ERROR: generate_glue: The output directory does not exist.
   At: modules/mono/editor/bindings_generator.cpp:1121.
```
This is an odd one.

**Comment by neikeq** [comment]
I can't reproduce it. Could it be that you are using this commit https://github.com/godotengine/godot/commit/a7c2ff5 or a commit after it and before this one https://github.com/godotengine/godot/commit/5fed954?

**Comment by cart** [comment]
I also suffer from this with clean builds on godotengine/godot@308a6f4f99a5204682a2612d097e5c066f88196f

**Comment by neikeq** [comment]
This is a regression from godotengine/godot#10883

**Comment by RameshRavone** [comment]
This only happens with relative path,

**Comment by neikeq** [comment]
Fixes by godotengine/godot#11011
